subsys/bluetooth/controller/hci/hci.c

```diff
@@ -1604,6 +1604,41 @@ static void le_big_terminate_sync(struct net_buf *buf, struct net_buf **evt,
 
 #if defined(CONFIG_BT_CONN)
 #if defined(CONFIG_BT_CENTRAL)
+
+static uint8_t check_cconn_params(bool ext, uint16_t scan_interval,
+				  uint16_t scan_window,
+				  uint16_t conn_interval_max,
```

Comment:
Should check conn_interval_min as well, even though it isn't used by the controller.

Reply:
Since it is not used at all by the controller I thought it was not worth it to be honest. At that point there is no risk of providing the controller with invalid data.

Reply:
Sure, no big deal, I just thought it would be more maintainable in case you change that to use the conn_interval_min instead, then you would have to update in this place. And also in case there is some conformance testing.